src/renderer/viz/expressions/basic/property.js

```diff
@@ -108,10 +109,10 @@ export default class Property extends BaseExpression {
     }
 
     _getMinimumNeededSchema () {
+        const type = aggregationTypes.UNKNOWN;
+
         return {
-            [this.name]: [{
-                type: 'unaggregated'
-            }]
+            [this.name]: [{ type }]
```

Comment:
Why have we changed this from 'unaggregated' to 'unknown'?

Reply:
Because there was a bug when checking the minimum needed schema in the windshaft class. Variables assigned to a property, by default, are unaggregated, but that's not true because we don't know that if what the variable represents is aggregated or not. We were doing the following to know if there were any aggregated property:

```js 
const aggregatedUsage = usages.some(x => x.type !== 'unaggregated');
```

But that's not correct, so I changed it to:

```js
const aggregatedUsage = usages.some(x => x.type === aggregationTypes.AGGREGATED);
```